libs/search-widget/src/widgets/search-widget/SearchBar.svelte

```diff
@@ -85,6 +85,7 @@
   export let json_schema = '';
   export let vectorset = '';
   export let chat_placeholder = '';
+  export let audit_metadata = '';
```

Comment:
should also be added to popup and chat widgets ?

Reply:
right, fixed!